script.js

```diff
@@ -0,0 +1,145 @@
+const addBox = document.querySelector(".add-box"),
+popupBox = document.querySelector(".popup-box"),
+popupTitle = popupBox.querySelector("header p"),
+closeIcon = popupBox.querySelector("header i"),
+titleTag = popupBox.querySelector("input"),
+descTag = popupBox.querySelector("textarea"),
+addBtn = document.querySelector(".button");
+
+
+var notes;
+if(localStorage.getItem("notes"))
+{
+    notes=JSON.parse(localStorage.getItem('notes'));
+}
+else{
+    notes=JSON.parse("[]");
+}
+let isUpdate = false, updateId;
```

Comment:
same here.
Please create variables separately in each line as it will be easier to maintain further.